src/plugins/image_display/ImageDisplay.hh

```diff
@@ -99,6 +99,9 @@ namespace plugins
     /// \brief Update from L_INT16
     private: void UpdateFromLInt16();
 
+    /// \brief Update from L_IN8T
+    private: void UpdateFromLInt8();
```

Comment:
Does adding this method to a header file in this case not break API/ABI since this is a plugin?

Reply:
These headers are not installed, so we don't need to worry about ABI.

But also note that adding a private function is ABI-safe. The problem is adding private member variables.